jobs/microk8s/clusterapi/release.sh

```diff
@@ -47,24 +53,69 @@ git clone https://github.com/canonical/cluster-api-control-plane-provider-microk
 #################################################################
 if [ "${RUN_TESTS}" = true ]
 then
-    echo "Run tests"
-    sudo snap install go --classic --channel=1.18/stable
+    docker login -u ${DOCKERHUB_USR} -p ${DOCKERHUB_PSW}
+
+
+    echo "Setup management cluster"
+    sudo lxc profile create microk8s || true
+    cat "${_DIR}/microk8s.profile" | sudo lxc profile edit microk8s
+
+    sudo snap install kubectl --classic || sudo snap refresh kubectl || true
```

Comment:
Is it possible that with the `|| true` we will endup with a setup without `kubectl`?